torchdynamo/variables/builder.py

```diff
@@ -126,7 +133,12 @@ def _common_constants():
             2048,
             4096,
             0.1,
+            0.01,
             0.5,
+            0.05,
+            352,  # Work around for Super_SloMo
```

Comment:
The new added above is common constants from torchbench, the bellow three lines are workaround for the two failed models.